images/filelogoffsetvolumeownership/entrypoint.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+# SPDX-FileCopyrightText: Copyright 2025 Dash0 Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+# See README.md for an explanation of this script and its purpose.
+
+# MAINTENANCE NOTE: THIS SCRIPT *DELIBERATELY* DOES NOT RUN WITH set -e! See README.md for more details.
+set -x
+
+/bin/mkdir -p /var/otelcol/filelogreceiver_offsets
+/bin/chown -R 65532:0 /var/otelcol/filelogreceiver_offsets
```

Comment:
This was previously in an inline command in internal/collectors/otelcolresources/desired_state.go.